core/templates/components/forms/schema-based-editors/schema-based-int-editor.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="input-group">
+  <span *ngIf="!expressionMode">
+    <mat-form-field class="long oppia-blue-on-focus md-default-theme m-0">
+      <input type="number" matInput [(ngModel)]="localValue" (change)="updateLocalValue()"
+             [disabled]="isDisabled" required [attr.focus-on]="labelForFocusTarget"
```

Comment:
There is no attribute called `focus-on`. It is a custom directive we made. Use `oppiaFocusOn` instead. To use that, you will have to import the directives module to schema-based editor module. `[attr.focus-on]="labelForFocusTarget"` -> `[oppiaFocusOn]="labelForFocusTarget"`. DO this in the other files as well.